modules/API/API.js

```diff
@@ -164,6 +165,12 @@ function initCommands() {
             }
             APP.store.dispatch(autoAssignToBreakoutRooms());
         },
+        'grant-moderator': participantId => {
+            if (!isLocalParticipantModerator(APP.store.getState())) {
```

Comment:
The caller of the API has no way to know they did anything wrong if we don't give an error...